pkg/deploy/util/util.go

```diff
@@ -228,6 +234,19 @@ func EncodeDeploymentConfig(config *deployapi.DeploymentConfig, codec runtime.Co
 	return string(bytes[:]), nil
 }
 
+func NewControllerRef(config *deployapi.DeploymentConfig) *metav1.OwnerReference {
+	blockOwnerDeletion := true
```

Comment:
@mfojtik I wish Go had a syntax for it ;) 

But these few lines are also from upstream.